Wikipedia/Code/MWKSection.m

```diff
@@ -275,6 +275,54 @@ - (NSString*)summary {
     }] componentsJoinedByString:@" "] wmf_summaryFromText];
 }
 
+static NSString* const WMFSectionDisambiguationTitlesXPathSelector = @"//div[@class='hatnote']//a/@href";
+
+- (nullable NSArray<MWKTitle*>*)disambiguationTitles {
+    NSArray* textNodes = [self elementsInTextMatchingXPath:WMFSectionDisambiguationTitlesXPathSelector];
+    if (!textNodes || !textNodes.count) {
```

Comment:
in fact, you don't need these checks at all. since the code below will just work if the textnodes is nil or the count is 0